src/app/pages/userslist/userslist.component.ts

```diff
@@ -1,12 +1,95 @@
-import { Component } from '@angular/core';
+import { Component, AfterViewInit, ViewChild, Input, Output, EventEmitter, ElementRef } from '@angular/core';
+import { BehaviorSubject, Observable, map, pipe } from 'rxjs';
+import { SharedModule } from '../../shared/shared.module';
+import { User } from '../../shared/models/user';
+import { UserService } from '../../services/user/user.service';
 
 @Component({
   selector: 'app-userslist',
   standalone: true,
-  imports: [],
+  imports: [ SharedModule ],
   templateUrl: './userslist.component.html',
   styleUrl: './userslist.component.css'
 })
-export class UserslistComponent {
+export class UserslistComponent implements AfterViewInit {
+  possibleNumbersOfUsers: number[] = [ 1, 2, 5, 10 ]
+  displayedColumns: string[] = [ 'id', 'userName', 'email', 'phoneNumber' ]
 
+  private _users = new BehaviorSubject<User[]>([]);
+  dataSource$: Observable<User[]> = this._users.asObservable();
+
+  constructor(private _userService: UserService) { }
+
+  @ViewChild('nextButton', { read: ElementRef }) nextButton!: ElementRef<HTMLButtonElement>;
+
+  @Input() numberOfUsers: number = this.possibleNumbersOfUsers[this.possibleNumbersOfUsers.length / 2];
```

Comment:
For Two-Way binding for Form, you don't need @Input() and @Output(). You just need your local variable for this, remove them